conda/environments/all_cuda-120_arch-x86_64.yaml

```diff
@@ -0,0 +1,76 @@
+# This file is generated by `rapids-dependency-file-generator`.
+# To make changes, edit ../../dependencies.yaml and run `rapids-dependency-file-generator`.
+channels:
+- rapidsai
+- rapidsai-nightly
+- dask/label/dev
+- conda-forge
+- nvidia
+dependencies:
+- c-compiler
+- cmake>=3.26.4
+- cuda-cudart-dev
+- cuda-nvcc=12.0
```

Comment:
I was going to comment that this shouldn't be versioned, but then I noticed that this doesn't match what dependencies.yaml says. I'm guessing this was fixed in dependencies.yaml during a PR review but dfg wasn't rerun yet. Rerunning locally and pushing should fix this (I also now see that CI is failing, I assume this is why).

Reply:
Yup, I suggested a fix in dependencies.yaml and now it's fixed here too. Resolving.